content/tutorials/reactor-event-cloudflare.textile

```diff
@@ -0,0 +1,256 @@
+---
+title: "Reactor Events Tutorial: Building A Realtime 2D Game with React"
+alt_title: "Building A Realtime 2D Game with React"
+excerpt: "Learn how to use our Reactor Functions to consume realtime data with Cloudflare"
+section: tutorials
+category: external-services
+index: 100
+platform: browser
+authors:
+- author_name: Abraham Jr. Agiri
+  author_bio: "I'm a software developer who loves writing and in love with open source"
+  author_profile_url: https://github.com/codeekage
+  author_image: ://avatars3.githubusercontent.com/u/24491408?s=400&v=4
+level: medium
+reading_time: 10
+languages:
+  - javascript
+  - nodejs
+ably_product: reactor
+tags:
+  - ably-features
+---
+
+The Ably Reactor provides a way to trigger events and to stream data from Ably's pub/sub channels. This turorial will go over how to use the Ably Reactor with "Cloudflare workers":https://developers.cloudflare.com/workers/quickstart/. 
+Imagine a delivery driver delivering parcels around the neighbourhood. Every time a parcel is delivered, we could trigger a serverless function to track and manage the time and date of delivery, to perhaps track how many more deliveries the driver has to make that day, even give the driver the location of the next delivery.
+
+h2. What we'll be building
+
+We'll be building a postal service delivery game using React and "Reactor Functions":https://www.ably.io/documentation/general/functions. Players use their arrow keys to drive a delivery van around a map to each of the postboxes that appear. Reactor events will be triggered every time the player 'delivers' a parcel to a postbox, this will call a Cloudflare worker which will notify a separate 'admin page' that the delivery has been made.
+
+
+<a href="/images/tutorials/reactor/reactor-event-cloudflare-game-screenshot.png" target="_blank">
+  <img src="/images/tutorials/reactor/reactor-event-cloudflare-game-screenshot.png" style="width: 100%" alt="screenshot of the game we will be making">
+</a>
+
+
+h2. Ably Reactor Prerequisites
+
+We're going to need an Ably API key for use by your browser application, and for the Cloudflare Worker, to allow it to post a response back to Ably. For our purposes, only one API key is required. If you extend the tutorial with your own functionality, you may wish to create a separate API key for the Cloudflare Worker if you need it to have a different set of permissions than those assigned for use by the browser app.
```

Comment:
Similar comments to https://github.com/ably/docs/pull/820/files, I believe a lot of this is "common" and should be in partials that probably already exist.

Reply:
I've created the partial necessary for setting up Reactor Rules in the IFTTT tutorial, it did not yet exist, but does now
I've swapped out the api creation text for a partial